src/core/service/notifications/jobs/handler.go

```diff
@@ -205,6 +206,23 @@ func (h *Handler) HandleRetentionTask() {
 			h.SendInternalServerError(err)
 			return
 		}
+
+		e := &event.Event{}
+		metaData := &metadata.RetentionMetaData{
+			Total:    retainObj.Total,
+			Retained: retainObj.Retained,
+			Deleted:  retainObj.Deleted,
+			Status:   "SUCCESS",
```

Comment:
always success?

Reply:
> always success?

Tag rentention info will be received only when task begin to  run.      when the task status with success received, there is no retention info with it.